samples/drivers/mbox/remote/sample.yaml

```diff
@@ -0,0 +1,9 @@
+sample:
+    name: MBOX IPC sample (remote)
+tests:
+    sample.drivers.mbox_remote:
+        platform_allow: nrf5340dk_nrf5340_cpunet
+        integration_platforms:
+          - nrf5340dk_nrf5340_cpunet
+        tags: mbox
+        build_only: true
```

Comment:
Why build_only? Why can't those test run? 

Maybe a readme is needed to explain how those samples are supposed to be used and how they fit into maybe something else, given that this is probably a multicore situation here.

Reply:
they can run but you need the hardware and you need to flash the local and remote cores. From the CI prospective how that can be managed?

Reply:
build_only is for unusable code that we build to verify builds, i.e. the `tests/drivers/build_all/` collection or footprint samples that  never run but try to just capture footprint data.
Until we have better support in twister to support testing multiple images, add

```
  harness: remote
```

This way we will know would need some remote support (that we are missing now). This will not run in CI becuase we do not support this harness yet.

Reply:
and, please consider adding some readme to those samples :)

Reply:
done